doc/vxlan/Overlay ECMP with BFD.md

```diff
@@ -452,5 +539,12 @@ The below cases are executed first for IPv4 and repeat the same for IPv6.
 |Create/Delete overlay routes to 4k with unique endpoints upto 4k | BFD Scaling  | Verify all 4k BFD sessions are succesfully created and sessions alive |
 
 ### 2.8.3 BGP advertising
-
-TBD
+The below cases are executed first for IPv4 and repeat the same for IPv6. 
+| Step | Goal | Expected results |
+|-|-|-|
+| Create a tunnel route and adevertise the tunnel route to all neighbor without community id | BGP | ALL BGP neighbors can recieve the advertised BGP routes |
+| Create a tunnel route and adevertise the tunnel route to all neighbor with community id | BGP | ALL BGP neighbors can recieve the advertised BGP routes with community id |
+| Update a tunnel route and adevertise the tunnel route to all neighbor with new community id | BGP | ALL BGP neighbors can recieve the advertised BGP routes with new community id |
+| Delete a tunnel route | BGP | ALL BGP neighbors can remove the previously advertised BGP routes |
+| Create 4k tunnel routes and adevertise all tunnel routes to all neighbor with community id | BGP scale | ALL BGP neighbors can recieve 4k advertised BGP routes with community id and record the time |
+| Updat BGP_PROFILE_TABLE with new community id for 4k tunnel routes and adevertise all tunnel routes to all neighbor with new community id | BGP scale | ALL BGP neighbors can recieve 4k advertised BGP routes with new community id and record the time |
```

Comment:
Lets also add a case to create the tunnel route first with a profile name, but the profile itself is created later.